migrations/20200408102501_add_project_profiles_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('project_profiles')
```

Comment:
`project_collaborators` would probably provide more context...

Reply:
I don't disagree, but since we are never going to query this table directly...